perses/rjmc/geometry.py

```diff
@@ -371,7 +371,7 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
         """
         _logger.info("Conducting forward proposal...")
         import copy
-        from perses.tests.utils import compute_potential_components
+        from perses.dispersed.utils import compute_potential_components
```

Comment:
Why is it that you think so? `dispersed/utils.py` does import a LOT of things (even scipy), I think it is worth it to keep this in the lower level. Maybe what we can do to make the readability a bit better is to have them in the top of the method (`_logp_propose` in this case). Does that sound okay?

Reply:
maybe we make a new utils file here:
https://github.com/choderalab/perses/tree/main/perses/utils
that is like hybrid_topology_utils or something
I figure if we are going to move this, lets move it to the right spot

Reply:
Yeah I agree `dispersed/utils.py` might not be the best place. I'm not actually sure what functionality `dispersed/` is meant to contain.

I agree that `perses/utils/` is a better place, maybe call it `energy_validation.py` instead of `hybrid_topology.py`, as the functions are used for generating htfs as well as geometry proposals.

Reply:
Does this sound reasonable? https://github.com/choderalab/perses/pull/1092#issuecomment-1226537892